source/statements.tex

```diff
@@ -276,7 +276,7 @@
 template<typename T, typename ... Rest> void g(T&& p, Rest&& ...rs) {
   // ... handle \tcode{p}
 
-  if constexpr (sizeof...(rs) > 0)
+  if constexpr (sizeof...(rs) > 0) {
```

Comment:
No, this isn't right; the closing brace matches the opening brace on line 276. It's the indent of what follows that's wrong, and the brace on line 293 is extraneous.